go/libkb/api.go

```diff
@@ -736,10 +704,6 @@ func (a *InternalAPIEngine) GetDecode(arg APIArg, v APIResponseWrapper) error {
 		return nil
 	}
 
-	if err := a.refreshSession(m, arg, reqErr); err != nil {
-		return err
-	}
-
```

Comment:
are these log messages about session refresh no longer accurate?

Reply:
With NISTs, there's never a reason to refresh a session.  The session is "always on" more or less, since the client gets to generate them whenever it wants, and it knows where they are stale.

Reply:
yes, i meant the next line which i can't comment on...the log message that says the session has been refreshed.

Reply:
ah ok... let me look!

Reply:
lines 743 and 835 in this panel, 707 and 795 in the other panel